### ISSUE TITLE
Excel Sheet Protection Inconsistency

### ISSUE THREAD

**Issue by calebw** [EPPlusSoftware/EPPlus#1792]
### EPPlus usage

Commercial use (I have a commercial license)

### Environment

Windows

### Epplus version

7.5.2

### Spreadsheet application

Excel

### Description

This bug is introduced in version 7.5.0.

Setup (or see attached):
Create an XLSX file.
Add arbitrary columns to its sheet.
Select all cells in the worksheet (aka click top left) > right click > Format Cells > Protection > **Uncheck** Locked.
Select the entire 1st row (the Header row) > right click > Format Cells > Protection > **Check** Locked.
Right Click the Sheet > Protect Sheet.
Save file.

Then write some basic EPPlus code to open this file and LoadFromCollection any arbitrary data into it. 

**Expected Behavior:**
The Header row remains locked and cannot be edited.
All other cells are editable. 

**Actual Behavior:**
All cells with values are locked. The Header as well as any loaded data. 

This behavior changed between versions 7.4.2 and 7.5.0.
I don't see any mention in the changelog that may be related to this nor any mention of breaking changes. 

Protection Password: "qwerty"
[ExcelIssueExample.xlsx](https://github.com/user-attachments/files/18215370/ExcelIssueExample.xlsx)
[sampleCode.txt](https://github.com/user-attachments/files/18215375/sampleCode.txt)

**Comment by JanKallman** [comment]
Yes,this is a bug. EPPlus seems to ignore the applyProtection attribute in this case. I will try to provide a fix.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.5.3
